.nvmrc

```diff
@@ -1 +1 @@
-8.11.2
\ No newline at end of file
+10.19.0
```

Comment:
@alexanderatallah updated the node version since node v8 is now outdated

Reply:
@taylorjdawson sounds good if it works now. there used to be a node-scrypt issue with 10 and 12. (also might be worth trying 12)